content-vs/dotnet-template-azure-iot-edge-module/.gitignore

```diff
@@ -0,0 +1,9 @@
+# Ignore the following files which are copied from ROOT/content/dotnet-template-azure-iot-edge-module/CSharp
+
+CSharp/Dockerfile*
```

Comment:
From the perspective of users, these files should be version controlled.

Reply:
this .ignore file is not used by user, it is just used to ignore files copied from content/

Reply:
Will this `.gitignore` file be copied to users' project folder?